src/webgpu/api/operation/sampling/anisotropy.spec.ts

```diff
@@ -6,14 +6,506 @@ Note that anisotropic filtering is never guaranteed to occur, but we might be ab
+  private vertexBuffer: GPUBuffer | undefined;
+  setup(): void {
+    this.pipeline = this.device.createRenderPipeline({
+      vertexStage: {
+        module: this.device.createShaderModule({
+          code: `
+            [[block]] struct Uniforms {
+                [[offset(0)]] matrix : mat4x4<f32>;
+            };
+
+            [[location(0)]] var<in> position : vec4<f32>;
+            [[location(1)]] var<in> uv : vec2<f32>;
+
+            [[set(0), binding(2)]] var<uniform> uniforms : Uniforms;
+
+            [[builtin(position)]] var<out> Position : vec4<f32>;
+            [[location(0)]] var<out> fragUV : vec2<f32>;
+
+            [[stage(vertex)]] fn main() -> void {
+                fragUV = uv;
+                Position = uniforms.matrix * position;
+            }
+            `,
+        }),
+        entryPoint: 'main',
+      },
+      fragmentStage: {
+        module: this.device.createShaderModule({
+          code: `
+            [[set(0), binding(0)]] var<uniform_constant> sampler0 : sampler;
+            [[set(0), binding(1)]] var<uniform_constant> texture0 : texture_2d<f32>;
+
+            [[builtin(frag_coord)]] var<in> FragCoord : vec4<f32>;
+
+            [[location(0)]] var<in> fragUV: vec2<f32>;
+
+            [[location(0)]] var<out> fragColor : vec4<f32>;
+
+            [[stage(fragment)]] fn main() -> void {
+                fragColor = textureSample(texture0, sampler0, fragUV);
+            }
+            `,
+        }),
+        entryPoint: 'main',
+      },
+      primitiveTopology: 'triangle-list',
+      colorStates: [{ format: 'rgba8unorm' }],
+      vertexState: {
+        vertexBuffers: [
+          {
+            arrayStride: 6 * Float32Array.BYTES_PER_ELEMENT,
+            attributes: [
+              {
+                // position
+                shaderLocation: 0,
+                offset: 0,
+                format: 'float4',
+              },
+              {
+                // uv
+                shaderLocation: 1,
+                offset: 4 * 4,
+                format: 'float2',
+              },
+            ],
+          },
+        ],
+      },
+    });
+
+    const matrixData = new Float32Array([
+      -1.7320507764816284,
+      1.8322050568049563e-16,
+      -6.176817699518044e-17,
+      -6.170640314703498e-17,
+      -2.1211504944260596e-16,
+      -1.496108889579773,
+      0.5043753981590271,
+      0.5038710236549377,
+      0,
+      -43.63650894165039,
+      -43.232173919677734,
+      -43.18894577026367,
+      0,
+      21.693578720092773,
+      21.789791107177734,
+      21.86800193786621,
+    ]);
+    this.uniformBuffer = this.device.createBuffer({
+      size: matrixData.byteLength,
+      usage: GPUBufferUsage.VERTEX | GPUBufferUsage.UNIFORM | GPUBufferUsage.COPY_DST,
+    });
+    this.device.defaultQueue.writeBuffer(this.uniformBuffer, 0, matrixData);
+
+    // position : vec4, uv : vec2
+    // uv is scaled
+    const vertexData = new Float32Array([
```

Comment:
(Code may get deleted, but) You could use prettier-ignore something like this:
```suggestion
    const vertexData = new Float32Array(/* prettier-ignore */ [
```
so you can format the block of data more nicely.